.github/config.yml

```diff
@@ -3,23 +3,18 @@
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
-  💖 Thanks for opening your first pull request! 💖
+  💖 **Thank you for opening your first pull request in this repository!** 💖
 
-  Please make sure you read the following:
+  A few things to keep in mind:
 
   * [Authorship Guidelines](https://github.com/fatiando/community/blob/main/AUTHORSHIP.md): Our rules for giving you credit for your contributions, including authorship on publications and [Zenodo](https://zenodo.org/communities/fatiando) archives.
   * [Code of Conduct](https://github.com/fatiando/community/blob/main/CODE_OF_CONDUCT.md): How we expect people to interact in our projects.
-
-  A few things to keep in mind:
-
-  * Remember to run `make format` to make sure your code follows our style guide.
-  * If you need help writing tests, take a look at the existing ones for inspiration. If you don't know where to start, let us know and we'll walk you through it.
-  * All new features should be documented. It helps to write the docstrings for your functions/classes before writing the code. This will help you think about your code design and results in better code.
-  * No matter what, we are really grateful that you put in the effort to do this! ⭐
+  
+  ⭐ **No matter what, we are really grateful that you put in the effort to do this!** ⭐
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
-  🎉🎉🎉 Congrats on merging your first pull request and welcome to the team! 🎉🎉🎉
+  🎉 **Congrats on merging your first pull request and welcome to the team!** 🎉
 
   **If you would like to be added as a author** on the [Zenodo](https://zenodo.org/communities/fatiando) archive of the next release, make sure that you have added your full name, affiliation, and [ORCID](https://orcid.org) (optional) to the `AUTHORS.md` file of this repository.
```

Comment:
Maybe we could add a little sentence telling people to feel free to open a new PR in case they forgot to add themselves as authors.

Reply:
Good catch! Done